cmd/psa/check_test.go

```diff
@@ -17,7 +17,7 @@ func Test_CheckCmd_ok(t *testing.T) {
 	err := afero.WriteFile(fs, "psatoken.cbor", testValidP2PSAToken, 0644)
 	require.NoError(t, err)
 
-	err = afero.WriteFile(fs, "es256.jwk", testValidKey, 0644)
+	err = afero.WriteFile(fs, "es256.jwk", testValidKeyPub, 0644)
```

Comment:
Ditto!

Reply:
same, the name here is irrelevant

Reply:
also in this case it's incidentally an ES256 key